components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -876,7 +876,9 @@ - (void)updatePlaceholderAlpha {
 }
 
 - (void)updatePlaceholderPosition {
-  self.placeholderTop.constant = _textInput.textInsets.top;
+  CGFloat diff = self.textInput.font.lineHeight - self.placeholderLabel.font.lineHeight;
```

Comment:
This looks unrelated to the change you described in the PR description. Is this fixing a different bug?
If it would be great to get it as a different PR.
If not, please explain more how the changes are linked together.

Reply:
It's an issue within the same setup that reproduced this bug - If I remove this the component would still be unusable. No problem with actually moving this to a new PR, but I wonder when will it be merged if not linked to an open issue.

Reply:
To be precise, the additional fix makes sure that placeholder text and texfield content are bottom aligned, instead of top aligned. The issue is only evident when point sizes of placeholder text and textfield are very different.